addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -62,6 +62,16 @@ class StockPickingBatch(models.Model):
         help=('Name of the batch from which this batch was derived')
     )
 
+    # This is a barcode and not a One2one to allow pallets that aren't
+    # in the system yet (because they are empty) to be reserved.
+    u_last_reserved_pallet_name = fields.Char(
```

Comment:
Do we want it readonly?

Reply:
It's now visible only for Trusted Users on the UI.